Chat App/ViewModel/Pub/PubChatLoginViewModel.swift

```diff
@@ -69,38 +71,39 @@ class PubChatLoginViewModel {
      - Parameter username: submitted user name
      */
     func loginUser(username: String) {
+        guard let hash = encodedPassword else { return }
```

Comment:
fatalError? for missing encodedPasssword?

Reply:
Removed return and added a fatal error in the else clause.